onnxruntime/__init__.py

```diff
@@ -13,6 +13,4 @@
 from onnxruntime.capi.session import InferenceSession, IOBinding
 from onnxruntime.capi import onnxruntime_validation
 
-from onnxruntime.capi.training import *
```

Comment:
Thanks. This is going to be handled in a separation discussion

> 
> 
> Need to add this back, otherwise `orttraining/orttraining/python/training/__init__.py` is not called, resulting in "no `TrainingParameters` error".

Thanks. This is going to be handled in a separation discussion